File: Selenium/Project/Activity2.java
package Project;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.AfterMethod;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity2 {

    WebDriver driver;

    @BeforeClass
    public void setup() {
        WebDriverManager.firefoxdriver().setup();
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "NULL");
        driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/crm");
    }

    @Test
    public void test1(){
        String src = driver.findElement(By.xpath("//img[@alt='SuiteCRM']")).getDomProperty("src");
        System.out.println(src);
    }

    @AfterClass
    public void tearDown(){
        driver.quit();
    }
}

File: Selenium/Project/Activity2.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service = service) as driver:

    # Open the browser to the URL
    driver.get("http://alchemy.hguy.co/crm")

    pageTitle = driver.title
    print(pageTitle)
    assert (pageTitle=="SuiteCRM")
    src = driver.find_element(By.XPATH,"//img[@alt='SuiteCRM']").get_attribute("src")
    print(src)
    assert src is not None

    driver.quit()